sdk/include/reactos/wine/test.h

```diff
@@ -354,6 +382,73 @@ int broken( int condition )
     ) && condition;
 }
 
+#ifdef USE_WHITELIST
+extern WHITELIST_FILE_ENTRY g_Whitelist[];
+extern int g_WhitelistLength;
+
+static PWHITELIST_ENTRY get_whitelist_entry(const char *File, int Line)
+{
+    int i, j;
+
+    for (i = 0; i < g_WhitelistLength; i++)
+    {
+        if (!strcmp(g_Whitelist[i].File, File))
+        {
+            PWHITELIST_ENTRY entries = g_Whitelist[i].Entries;
+            for (j = 0; j < g_Whitelist[i].NumEntries; j++)
+            {
+                if (entries[j].Line == Line)
+                {
+                    return &entries[j];
+                }
+            }
+        }
+    }
+
+    return NULL;
+}
+
+static int does_whitelist_entry_apply(PWHITELIST_ENTRY entry)
+{
+    if ((entry->Attributes & WLA_BROKEN_ROS) && (strcmp(winetest_platform, "reactos") == 0))
+    {
+        return 1;
+    }
+
+    unsigned int winver = GetVersion();
```

Comment:
This will not get above 0x602 without a manifest, so better to use `RtlGetVersion`

![image](https://user-images.githubusercontent.com/610685/127217672-42e529fb-7a71-4f05-a73a-49e23f4f05fc.png)